public/styles.css

```diff
@@ -729,3 +730,69 @@ td {
 	padding: 2%;
 	text-align: center;
 }
+#QATitle {
```

Comment:
Not a huge deal, but for CSS I think the convention is lowercase-hyphenated, not camelCase... Alice told me when I have just a few classes/IDs defined so it wasn't a huge effort to update it, so if you have a lot, probably not worth updating.

Reply:
Went ahead and updated those since I didn't have too many tags to change. I'll also keep this in mind when writing more css tags in the future.